packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/GroundOverlaySink.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.googlemaps;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import com.google.android.gms.maps.model.BitmapDescriptor;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.LatLngBounds;
+
+/** Receiver of GroundOverlayOptions configuration. */
+interface GroundOverlaySink {
+  void setTransparency(float transparency);
```

Comment:
Can you add javadocs to these methods for where someone can learn more about the inputs either here or in groundOverlayController

For example what are the min and max values for transparency. Can zindex be negative? what does setClickable do? etc.